Dockerfile

```diff
@@ -6,8 +6,6 @@ WORKDIR /usr/src/app/
 COPY engine.json /
 COPY package.json /usr/src/app/
 
-ENV NODE_ENV production
```

Comment:
Hmm, how were our circle builds passing before if `devDependencies` weren't installed? I guess this env var had no effect?